user_tools/src/spark_rapids_tools/cmdli/tools_cli.py

```diff
@@ -63,23 +61,23 @@ def qualification(self,
                 cluster name on the CSP.
         :param cluster: The CPU cluster on which the Spark application(s) were executed.
                Name or ID (for databricks platforms) of cluster or path to cluster-properties.
-        :param platform: defines one of the following "onprem", "emr", "dataproc", "dataproc-gke",
+        :param platform: Defines one of the following: "onprem", "emr", "dataproc", "dataproc-gke",
                "databricks-aws", and "databricks-azure".
-        :param output_folder: path to store the output
+        :param output_folder: Local or HDFS path to store the output.
```

Comment:
I think this should just be Local right now until we fix HDFS from python

Reply:
+1